hw/ip/otp_ctrl/doc/otp_ctrl_mmap.md

```diff
@@ -3,79 +3,98 @@ DO NOT EDIT THIS FILE DIRECTLY.
+|         |                           |            |      32bit       |                    ROT_CREATOR_AUTH_CODESIGN_ECDSA_KEY3                     |     0x544      |     68     |
+|         |                           |            |      32bit       |                     ROT_CREATOR_AUTH_CODESIGN_SPX_KEY0                      |     0x588      |     40     |
+|         |                           |            |      32bit       |                     ROT_CREATOR_AUTH_CODESIGN_SPX_KEY1                      |     0x5B0      |     40     |
+|         |                           |            |      32bit       |                     ROT_CREATOR_AUTH_CODESIGN_SPX_KEY2                      |     0x5D8      |     40     |
+|         |                           |            |      32bit       |                     ROT_CREATOR_AUTH_CODESIGN_SPX_KEY3                      |     0x600      |     40     |
+|         |                           |            |      32bit       |                ROT_CREATOR_AUTH_CODESIGN_BLOCK_SHA2_256_HASH                |     0x628      |     32     |
+|         |                           |            |      64bit       | [ROT_CREATOR_AUTH_CODESIGN_DIGEST](#Reg_rot_creator_auth_codesign_digest_0) |     0x648      |     8      |
+|    4    |  ROT_CREATOR_AUTH_STATE   |     40     |      32bit       |                      ROT_CREATOR_AUTH_STATE_ECDSA_KEY0                      |     0x650      |     4      |
+|         |                           |            |      32bit       |                      ROT_CREATOR_AUTH_STATE_ECDSA_KEY1                      |     0x654      |     4      |
+|         |                           |            |      32bit       |                      ROT_CREATOR_AUTH_STATE_ECDSA_KEY2                      |     0x658      |     4      |
+|         |                           |            |      32bit       |                      ROT_CREATOR_AUTH_STATE_ECDSA_KEY3                      |     0x65C      |     4      |
+|         |                           |            |      32bit       |                       ROT_CREATOR_AUTH_STATE_SPX_KEY0                       |     0x660      |     4      |
+|         |                           |            |      32bit       |                       ROT_CREATOR_AUTH_STATE_SPX_KEY1                       |     0x664      |     4      |
+|         |                           |            |      32bit       |                       ROT_CREATOR_AUTH_STATE_SPX_KEY2                       |     0x668      |     4      |
+|         |                           |            |      32bit       |                       ROT_CREATOR_AUTH_STATE_SPX_KEY3                       |     0x66C      |     4      |
+|         |                           |            |      64bit       |    [ROT_CREATOR_AUTH_STATE_DIGEST](#Reg_rot_creator_auth_state_digest_0)    |     0x670      |     8      |
```

Comment:
Just double checking here: what is the point of the digest if this partition needs to remain unlocked? Is it just automatically created by the autogen system?

Reply:
This partition was specified to have a locking digest in the RFC. We can remove it, but it also does not hurt to just leave it there - this is really up to you. If you think this should never be lockable, we can remove it.

@moidx

Reply:
SG, I suppose it could be useful to be able to lock further key authorizations permanently.